### DISCUSSION TITLE
Collaborating

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#208] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Hey, @3kh0 , I've noticed I can't mark answers anymore. Is this on purpose?

**Comment by 3kh0** [comment, marked as answer]
@yeeteeyt I did not change anything... see if you can mark this as the answer

**Reply by ghost** [reply]
Oh, ok. Guess github is being weird again... Sorry!
